### PR TITLE
Feat: Display a single armor option as checkbox instead of radioset.

### PR DESCRIPTION
**Pull request by TheMost82**
Fixes not being able to deselect a single armor option.

### CODE REVIEW

**Comment by TheMost82** [comment]
Confirmed working example:
image
image

Still working for existing multiple selection units:
image

**Comment by TheMost82** [comment]
Thanks in advance for reviewing!
One sidenote: Now that i got the dev env up and running i am thinking about refactoring the whole unit section a bit to make all radio sets and checkboxes easier to read.
Would require a larger review then though...  :)

File: src/pages/unit/Unit.js [hunk 1]
```diff
@@ -706,8 +714,37 @@ export const Unit = ({ isMobile }) => {
...
           </>
+        )}           
+        {unit.armor && unit.armor.length === 1 && (
+          <>
+            <h2 className="unit__subline">
+              <FormattedMessage id="unit.armor" /> 
+            </h2>
+            {unit.armor.map(
+              ({ points, perModel, id, active = false, ...equipment }) => (    
+                <div className="checkbox" key={id}>
```
[the remark this answers is not in the document]
Comment (TheMost82) [reply src/pages/unit/Unit.js:725]: I will test it in my next session to check how it looks and feels.
Also i got an idea regarding the active state and being able to remove a mandatory option.
Will update the PR accordingly after my next session :)

Dont worry about the timing, everything < 1 Week works well for me :)

Happy painting and may the dice gods be in your favour!

**Comment by TheMost82** [comment]
Alright, finally got back to this topic, were a few hectic weeks and did not have the mind for it :)
Refactored based on your comments:
- Less duplicated code, now decides inline if checkbox or radio rendering should be used

I also tried to disable the checkbox when only a single armor option is available that is already activated by default in the units source -  but I did not get this to work, only based on the already loaded units armor where the current active state is toggled.
Maybe you could add this and show me how one would do that? 🥺

**Comment by nthiebes** [comment]
I merged and deployed your changes! Besides some adjustments to datasets (removing a few "no armour" options) I made sure to use radio buttons for single armor options that are active by default: https://github.com/nthiebes/old-world-builder/commit/e485e18737641cade6b86891b52d52cb2201f709

**Comment by TheMost82** [comment]
Thanks for sharing, that helped my understanding of the project structure! 👍

**Pull request merged**